src/app/bav/controllers/root.js

```diff
@@ -0,0 +1,21 @@
+const { Controller: BaseController } = require("hmpo-form-wizard");
+
+class RootController extends BaseController {
+  async saveValues(req, res, next) {
+    const sharedClaims = req.session?.shared_claims;
+
+    if (sharedClaims) {
+
+      if (sharedClaims?.name?.length > 0) {
+        req.sessionModel.set("firstName", sharedClaims.name[0].nameParts[0].value);
+        req.sessionModel.set("surname", sharedClaims.name[0].nameParts[2].value);
+      }
+      if (sharedClaims?.birthDate?.length > 0) {
```

Comment:
Do date of birth might come in in the shared claims, but we don't actually ask a user for it - is that correct?

Reply:
Clarified with @mudassar-jafer that DoB is not expected in BAV - have removed from shared claims